config/jobs/kubernetes/sig-storage/sig-storage-gce-config.yaml

```diff
@@ -181,7 +181,7 @@ presubmits:
         - --gcp-node-image=ubuntu
         - --image-family=ubuntu-2004-lts
         - --image-project=ubuntu-os-cloud
-        - --env=KUBE_CONTAINER_RUNTIME=docker
+        - --env=KUBE_CONTAINER_RUNTIME=containerd
```

Comment:
```suggestion
        - --env=KUBE_CONTAINER_RUNTIME=containerd
        - --env=KUBE_UBUNTU_INSTALL_CONTAINERD_VERSION=v1.5.9                                                                                                               
        - --env=KUBE_UBUNTU_INSTALL_RUNC_VERSION=v1.0.3
```

Reply:
@endocrimes Thanks for the help. I had one question - 
- If by specifying the versions of the container runtime helps in bumping their versions quickly after releases, then why docker's version was not specified before.

Reply:
@NikhilSharmaWe With docker the version just came from the node image, with containerd we started testing versions more explicitly.